docs/CONFIGURATION.md

```diff
@@ -28,6 +28,30 @@ Url of the [IPFS](https://ipfs.io) gateway to load APM repos from. If you intend
 
 Which source to load app frontend assets from. Can be one of `ipfs` (uses the configured IPFS gateway) or `local` (local development servers for each app, running on `localhost:300x`). If you intend to serve assets from a local IPFS daemon, you should set this to `ipfs`.
 
+It is also possible to define how every app is loaded individually using a comma-separated list, which contains a series of locations defined as `<app ID>:<app location>`.
+
+```console
+ARAGON_APP_LOCATOR=0xbf8491150dafc5dcaee5b861414dca922de09ccffa344964ae167212e8c673ae:http://localhost:1234,0xbf8491150dafc5dcaee5b861414dca922de09ccffa344964ae167212e8c673ae:http://localhost:3333 npm start
+```
+
+Individual app locators can also its “known name” if it exist. Known names are `Agent`, `Finance`, `Fundraising`, `Survey`, `TokenManager`, `Vault`, and `Voting`. When a known name is used without any location, they will get fetched them from their [assigned local ports](src/known-app-ids.js), which are used by their respective development servers (see [aragon-apps](https://github.com/aragon/aragon-apps)).
```

Comment:
```suggestion
Individual app locators can also use its “known name” if it exists. Known names are `Agent`, `Finance`, `Fundraising`, `Survey`, `TokenManager`, `Vault`, and `Voting`. When a known name is used without any location, they will get fetched them from their [assigned local ports](src/known-app-ids.js), which are used by their respective development servers (see [aragon-apps](https://github.com/aragon/aragon-apps)).
```

Ah, I just realized we could also detect ENS names, so you could use the "official" app names (e.g. `voting.aragonpm.eth`) instead of a raw hash.

Reply:
Ah yes totally!

Reply:
Done in 3fe882c813ab0aa2d4bea5e69305b0fba6baa6e3